### DISCUSSION TITLE
Questions about model training

### DISCUSSION THREAD

**Discussion by kadirnar** [TheMistoAI/MistoControlNet-Flux-dev#22] [Q&A] [answered]
1. Will you share the train code?
2. How many A100s did you use to train the model?
3. Can you give information about the dataset size?

**Comment by baizh0u** [comment, marked as answer]
> 1. Will you share the train code?
> 
> 2. How many A100s did you use to train the model?
> 
> 3. Can you give information about the dataset size?

1. Not this time.
2. We have done a lot of experiments, it depends on how many parameters the model has, we try 1-4 A100-80gb
3. The more the better.
